### PR TITLE
document missing `apt` dependency handling

### PR DESCRIPTION
**Pull request by s-hertel**
##### SUMMARY

Related #82485

Document `apt` module dependency handling

##### ISSUE TYPE

- Docs Pull Request

### CODE REVIEW

File: lib/ansible/modules/apt.py [hunk 1]
```diff
@@ -213,6 +213,8 @@
    - When used with a C(loop:) each package will be processed individually, it is much more efficient to pass the list directly to the O(name) option.
    - When O(default_release) is used, an implicit priority of 990 is used. This is the same behavior as C(apt-get -t).
    - When an exact version is specified, an implicit priority of 1001 is used.
+   - If the interpreter can't import ``apt`` the module will check in system-owned interpreters and attempt to install it if it's not found.
```
Comment (s-hertel) [inline lib/ansible/modules/apt.py]:
```suggestion
   - If the interpreter can't import ``python3-apt``/``python-apt`` the module will check in system-owned interpreters and attempt to install the dependency. If the dependency is installed successfully, the module will be respawned.
```

File: lib/ansible/modules/apt.py [hunk 2]
```diff
@@ -213,6 +213,8 @@
...
+     If successful, the module will be respawned with the interpreter containing the library.
```
Comment (s-hertel) [inline lib/ansible/modules/apt.py]:
```suggestion
```

File: lib/ansible/modules/apt.py [hunk 3]
```diff
@@ -1257,7 +1258,7 @@ def main():
         #    made any more complex than it already is to try and cover more, eg, custom interpreters taking over
         #    system locations)
 
-        apt_pkg_name = 'python3-apt' if PY3 else 'python-apt'
```
Comment (bcoca) [inline lib/ansible/modules/apt.py]: We might want to make this change in diff PR as this breaks existing 2.7 clients (which we break elsewhere i know, but people can still copy back the apt module itself to older ansible).

Comment (s-hertel) [reply lib/ansible/modules/apt.py]: :+1: Fixed.

**Pull request merged**